packages/model-viewer/src/three-components/SmoothControls.ts

```diff
@@ -764,9 +783,19 @@ export class SmoothControls extends EventDispatcher {
       element.removeEventListener('pointermove', this.onPointerMove);
       element.removeEventListener('pointerup', this.onPointerUp);
       element.removeEventListener('touchmove', this.disableScroll);
-      if (this.enablePan) {
-        this.recenter(event);
+
+      /** Something for double click detection.
+       * Only fire the recenter event on double click. This prevents errant (single) click events (such as giving back focus to model-viewer) from changing the current view.
+       * If the time difference between now and the previous mouse click (not the current one, the one before that) is less than TAP_MS then call it a double click.
+      */
```

Comment:
This is interesting, but appears to be unrelated to the rest of this PR. Perhaps separate it? This is already a bit large.